src/java.base/share/classes/java/lang/System.java

```diff
@@ -813,6 +814,10 @@ public static native void arraycopy(Object src,  int  srcPos,
      * Note that even if the security manager does not permit the
      * {@code getProperties} operation, it may choose to permit the
      * {@link #getProperty(String)} operation.
+     * <p>
+     * Additional locale-related system properties defined by the
+     * {@link Locale##default_locale Default Locale} section in the {@code Locale}
```

Comment:
Is that "##" a typo? double pound signs usually leads to the top of the page. I see PR 17065 also had "##" for links to default_locale.

Reply:
Thanks, Joe. As Roger mentioned, the `##` can be used for user-defined anchors since JDK20 ( https://bugs.openjdk.org/browse/JDK-8294195)